### DISCUSSION TITLE
How to make a `inverted` menu without "inverting" the dropdown button?

### DISCUSSION THREAD

**Discussion by batata004** [fomantic/Fomantic-UI#1788] [Elements (button, image, label, loader, ...)] [answered]
Please take a look at https://jsfiddle.net/y9a72Lsj/

Click in the left button and in the right. I would like to have the menu inverted (dark background) but I would like the dropdown button to not be inverted. I cant find a way to do that because everytime I add the `inverted` class to the `.dropdown` element, everything gets inverted (the button itself and the menu). I tried adding only `inverted` class to the `.menu` but didnt work. 

I know I can override the `.ui.dropdown .menu` class to change the background but I would like to know if I there is a way to do this without any "hack"? If not possible, I would suggest a feature request to allow the menu element to be inverted with the `.inverted` class added to it!

**Comment by lubber-de** [comment]
I'll prepare a PR for support of `.inverted.menu`

**Comment by lubber-de** [comment, marked as answer]
#1793 implements support for `inverted menu` inside non-inverted dropdowns
See your adjusted jsfiddle here https://jsfiddle.net/lubber/v2mc7gre/2

**Comment by batata004** [comment]
Awesome news!!! That works perfectly, pretty nice of you :) I will wait the next release to grab this change from the CDN :)
